app/views/responses/show.html.haml

```diff
@@ -1,8 +1,30 @@
-%tr
-	%td= link_to "Edit", edit_response_path(@response)
-	%td
-		= form_for @response, url: response_path(@response.id), html: {:onsubmit => "return confirm('Are you sure you want to delete the question?');"}, method: :delete do |f|
-			= submit_tag('Delete')
-%p= @response.text
-%p= @response.user_id
-%p= (@response.created_at).strftime("%m-%d-%Y")
\ No newline at end of file
+%html{:lang => "en"}
+  %head
+    %title Bootstrap Case
+    %meta{:charset => "utf-8"}/
+    %meta{:content => "width=device-width, initial-scale=1", :name => "viewport"}/
+    %link{:href => "http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css", :rel => "stylesheet"}/
```

Comment:
Bootstrap and other javascript/css libraries shouldn't be included here. They are handled as ruby gems and should already be available. Including libraries here could cause conflicts.